server/service/src/sync/test/integration/transfer/mod.rs

```diff
@@ -3,12 +3,9 @@ mod returns;
 mod shipments;
 use super::{
     central_server_configurations::{ConfigureCentralServer, SiteConfiguration},
-    SyncIntegrationContext,
-};
-use crate::{
-    service_provider::ServiceProvider,
-    sync::{synchroniser::Synchroniser, test::integration::init_test_context},
+    create_site, init_test_context, FullSiteConfig,
```

Comment:
In this file it's slightly different, needed to keep SiteContext, because it was easier to have SiteContext and the processors in a tuple, when using in tests (since SiteContext will go into callback but processor runs after callback, otherwise ownership issues and a of restructuring)